modules/odc-creating-apiserversource.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * applications/application_life_cycle_management/odc-knative-event-sources.adoc
+
+[id="odc-creating-apiserversource_{context}"]
+= Creating an ApiServerSource event source using the Developer perspective
+
+.Procedure
+
+. Navigate to the **Add** page.
++
+image::event-sources-odc.png[Add an Event Source in the Developer perspective]
```

Comment:
Suggestion: maybe drop this image. It keeps getting updated and becomes a maintenance overhead. For instance in 4.6 they will be adding a Channel card and you will need to update it.